### PR TITLE
Register Subdomain: marioannier

### PR DESCRIPTION
**Pull request by marioannier**
Added the domain: marioannier

### CODE REVIEW

File: domains/marioannier.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "Marioagp",
+        "email": "mario.annier@gmail.com"
+      },
+
+      "record": {
+        "A": ["marioannier"]
```
Comment (WilliamDavidHarrison) [inline domains/marioannier.json:9]: Invalid record.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/marioannier.json (hunk 1 above)
Comment (marioannier) [reply domains/marioannier.json:9]: Is not correct "username": "Marioagp",
I think is "username": "marioannier",

File: domains/marioannier.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "Marioagp",
```
Comment (WilliamDavidHarrison) [inline domains/marioannier.json:4]: Please change this to your GitHub username.

File: domains/marioannier.json (hunk 1 above)
Comment (WilliamDavidHarrison) [inline domains/marioannier.json:9]: Please add a valid IP address here.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
No response

**Pull request closed without merging**